src/js/components/common/dialogs/deviceconnectiondialog.js

```diff
@@ -73,7 +73,7 @@ export class DeviceConnectionDialog extends React.Component {
               </p>
             </ReactTooltip>
           </div>
-          <p>The steps are optimized for Linux workstations and devices running an OS in the Debian family such as Raspbian, Ubuntu and Debian.</p>
+          <p>The steps are optimized for Linux workstations and Raspberry Pi devices, but can be used on any ARM device running a Debian family OS.</p>
```

Comment:
If we get rid of the "generic ARM" I would change this whole section to something like this:

----

During the guided evaluation we will walk you through installing Mender on a 
device and deploying:

* a simple application update
* a full system update.
* a Docker container update

The evaluation of hosted Mender is optimized for Raspberry Pi 3 & 4 running 
Raspberry Pi OS. If you have such a device at hand you can proceed to prepare
and connect your device.

    < Update images to only contain Raspbian + maybe a picture of the RPi SBC >

    < PREPARE AND CONNECT MY RASPBERRY PI >

If you don't have a Raspberry Pi, you can use a Docker-run virtual device to try out Mender

    < I don't have my own device - use a virtual device for now >

Reply:
To further elaborate, I think the first pop-up in the UI should me very similar to this:

https://docs.mender.io/development/get-started/preparation

Above is the "landing page" if your start position is in the docs. This was at least what I had in mind.

Reply:
it should now be very close to the way the landing page in the docs is set up - the only concern I have might be that users with other devices feel a bit excluded, but with the changed phrasing for the virtual device and the popularity of Rpi we should be good 👍

Reply:
> the only concern I have might be that users with other devices feel a bit excluded

I understand this concern, but I think bringing custom (non-optimized) devices in to the on-boarding process is to complicated. Our reference environment is based on RPi and QEMU, this you use to test Mender use-cases (system, application, docker updates), it will work because we ensure it works :).

After you are finished with the on-boarding, then it is time to install on your custom device with the knowledge you acquired during onboarding :)